tron/config/config_parse.py

```diff
@@ -583,10 +583,20 @@ class ValidateMesos(Validator):
     option = True
     defaults = {
         'enabled': False,
+        'default_volumes': (),
+        'dockercfg_location': None,
```

Comment:
I think the default got lost here. Is this where the `file:///` goes or is that piece of configuration moving somewhere else?

Reply:
Was planning to get it from system paasta config (https://github.com/Yelp/paasta/pull/1868) but I could put file:///root/.dockercfg back as a default if that's a fairly standard value.